ui/src/app/Custom Validator/ConfirmPass.validator.ts

```diff
@@ -0,0 +1,18 @@
+import { FormGroup } from '@angular/forms';
```

Comment:
is this function will work with any controls or just password

Reply:
for any two controls you want to compare them?